src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -103,48 +103,27 @@ def string_to_measurement_model(
 
         # This dict will be made much smaller once "probsolve_ivp" is leaned out.
         choose_meas_model = {
-            "EKF0": pnfs.DiscreteEKFComponent.from_ode(
+            "EK0": pnfs.DiscreteEKFComponent.from_ode(
                 ivp,
                 prior=prior,
                 ek0_or_ek1=0,
                 evlvar=measurement_noise_covariance,
                 forward_implementation="sqrt",
                 backward_implementation="sqrt",
             ),
-            "EKS0": pnfs.DiscreteEKFComponent.from_ode(
-                ivp,
-                prior=prior,
-                ek0_or_ek1=0,
-                evlvar=measurement_noise_covariance,
-                forward_implementation="sqrt",
-                backward_implementation="sqrt",
-            ),
-            "EKF1": pnfs.DiscreteEKFComponent.from_ode(
-                ivp,
-                prior=prior,
-                ek0_or_ek1=1,
-                evlvar=measurement_noise_covariance,
-                forward_implementation="sqrt",
-                backward_implementation="sqrt",
-            ),
-            "EKS1": pnfs.DiscreteEKFComponent.from_ode(
+            "EK1": pnfs.DiscreteEKFComponent.from_ode(
                 ivp,
                 prior=prior,
                 ek0_or_ek1=1,
                 evlvar=measurement_noise_covariance,
                 forward_implementation="sqrt",
                 backward_implementation="sqrt",
             ),
-            "UKF": pnfs.DiscreteUKFComponent.from_ode(
-                ivp,
-                prior,
-                evlvar=measurement_noise_covariance,
-            ),
-            "UKS": pnfs.DiscreteUKFComponent.from_ode(
-                ivp,
-                prior,
-                evlvar=measurement_noise_covariance,
-            ),
+            # "UK": pnfs.DiscreteUKFComponent.from_ode(
+            #     ivp,
+            #     prior,
+            #     evlvar=measurement_noise_covariance,
+            # ),
```

Comment:
do we want the UKF in this convenience function or not? Pro: it is so easy to do, contra: if we dont include it here, we can refer value checks from probsolve_ivp to this function (and it would be more in line with us telling people: dont use the ukf?)